aten/src/ATen/native/cuda/UnaryFractionKernels.cu

```diff
@@ -117,17 +117,6 @@ __host__ __device__ static inline double nearbyint_wrapper(double a) {
   return ::nearbyint(a);
 }
 
-__host__ __device__ static inline c10::complex<float> nearbyint_wrapper(c10::complex<float> a) {
```

Comment:
I wonder if we need to add this back.

@cpuhrsch generously fixed many of the internal build issues, but clangtidy is complaining about a slew of intrinsic issues like

```
use of undeclared identifier '__builtin_ia32_reduce_fmul_pd512'; did you mean '__builtin_ia32_vpermilpd512'?
../../../../mnt/gvfs/third-party2/llvm-fb/6dee269d7fd86fcf635dc05968a63300f6c89fc7/12/platform009/d3a69bd/lib/clang/stable/include/avx512fintrin.h:9316:10: '__builtin_ia32_vpermilpd512' declared here
    return __builtin_ia32_reduce_mul_q512(__W);
           ^
```

Reply:
Hmm. Will add it back.

Thanks @cpuhrsch!